frontend/components/CalleeTable.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+    <table class="Table">
+        <tr>
+            <th class="Heading">Callee Contract</th>
+            <th class="Heading">Address</th>
+        </tr>
+        <tr v-for="[callee, address] of Object.entries(callees)" :key="callee">
```

Comment:
What do we display if there are no callees (ie, empty object is provided)?
Please handle this properly and add a story for this case

Reply:
Added `not found` text as full row:

<img width="645" alt="image" src="https://user-images.githubusercontent.com/47861849/152796216-c4a4acc6-7616-4a98-baf3-ace2cc966087.png">

Is the text fine? 🙂